nntrainer/include/layers.h

```diff
@@ -149,7 +149,7 @@ class Layer {
    * @retval #ML_ERROR_NONE Successful.
    * @retval #ML_ERROR_INVALID_PARAMETER invalid parameter.
    */
-  virtual int setProperty(const char *key, const char *value) = 0;
+  virtual int setProperty(const char *key, std::vector<std::string> values) = 0;
```

Comment:
`@param[in] value value of property` comment should be updated.
* value -> values

Reply:
yes. Thanks.!